docs/index.rst

```diff
@@ -1,5 +1,5 @@
 ..
-  Copyright 2020�2022 Intel Corporation
+  Copyright 2020�2022 Intel Corporation
```

Comment:
I've seen this weird diff before. I there is a character that should be replaced by a `-` here.

Reply:
Addressed in [90ce137](https://github.com/oneapi-src/oneMKL/pull/510/commits/90ce13728ac9a2a6a4ed4f969ca1ccb9de50e34b), along with updating copyright dates to 2022-2024.